### PR TITLE
docs: add Airbyte integration guide

### PR DESCRIPTION
**Pull request by TC-MO**
Create new Airbyte integration guide
add new images
reorder the integrations in the sidebar

### CODE REVIEW

File: sources/platform/integrations/airbyte.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Airbyte
+sidebar_lbel: Airbyte
+description: Learn how to integrate your Apify Datasets with Airbyte.
```
Comment (TheoVasilis) [inline sources/platform/integrations/airbyte.md:4]:
```suggestion
description: Learn how to integrate your Apify datasets with Airbyte.
```

File: sources/platform/integrations/airbyte.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+sidebar_position: 11.15
+slug: /integrations/airbyte
+---
+
+**Learn how to integrate your Apify Datasets with Airbyte.**
```
Comment (TheoVasilis) [inline sources/platform/integrations/airbyte.md:9]:
```suggestion
**Learn how to integrate your Apify datasets with Airbyte.**
```

File: sources/platform/integrations/airbyte.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+---
+
+Airbyte is an open-source data integration platform that allows you to move your data between different sources and destinations using pre-built connectors, which are maintained either by Airbyte itself or by its community.
+One of these connectors is the Apify Dataset connector, which makes it simple to move data from Apify Datasets to any supported destination.
```
Comment (TheoVasilis) [inline sources/platform/integrations/airbyte.md:14]:
```suggestion
One of these connectors is the Apify dataset connector, which makes it simple to move data from Apify datasets to any supported destination.
```

File: sources/platform/integrations/airbyte.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Once you have all the necessary accounts set up, you need to set up the Apify connector.
+To do so, you will need to navigate to **Sources** tab in Airbyte and select **Apify Dataset**
+
+![Airbyte sources tab](./images/airbyte-sources.png)
+
+You will need to provide a **dataset ID** and your Apify API Token. You can find both of these in [Apify Console](https://console.apify.com).
+
+![Airbyte source setup](./images/airbyte-source-setup.png)
+
+To find your **dataset ID**, you need to navigate to **Storage** tab in Apify Console. Copy it and paste it in Airbyte.
```
Comment (TheoVasilis) [inline sources/platform/integrations/airbyte.md:32]:
```suggestion
To find your **dataset ID**, you need to navigate to the **Storage** tab in Apify Console. Copy it and paste it in Airbyte.
```

File: sources/platform/integrations/airbyte.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+![Datasets in app](./images/datasets-app.png)
+
+To find your Apify API Token, you need to navigate to **Settings** tab and select **Integrations**. Copy it and paste it in the relevant field in Airbyte.
```
Comment (TheoVasilis) [inline sources/platform/integrations/airbyte.md:36]:
```suggestion
To find your Apify API token, you need to navigate to the **Settings** tab and select **Integrations**. Copy it and paste it in the relevant field in Airbyte.
```

File: sources/platform/integrations/airbyte.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+![Integrations token](./images/apify-integrations-token.png)
+
+And that's it! You now have Apify Datasets set up as a Source, and you can use Airbyte to transfer your datasets to one of the available destinations.
```
Comment (TheoVasilis) [inline sources/platform/integrations/airbyte.md:40]:
```suggestion
And that's it! You now have Apify datasets set up as a source, and you can use Airbyte to transfer your datasets to one of the available destinations.
```

File: sources/platform/integrations/airbyte.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+To learn further steps on how to setup a Connection, visit [Airbyte's documentation](https://docs.airbyte.com/using-airbyte/getting-started/set-up-a-connection)
```
Comment (TheoVasilis) [inline sources/platform/integrations/airbyte.md:42]:
```suggestion
To learn more about how to setup a connection, visit [Airbyte's documentation](https://docs.airbyte.com/using-airbyte/getting-started/set-up-a-connection)
```

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

File: sources/platform/integrations/airbyte.md (hunk 7 above)
Comment (TC-MO) [reply sources/platform/integrations/airbyte.md:42]: Shouldn't this be capitalized as is in Airbyte docs?

File: sources/platform/integrations/airbyte.md (hunk 6 above)
Comment (TC-MO) [reply sources/platform/integrations/airbyte.md:40]: Similar ask here, if Airbyte capitalizes Source shouldn't we as well?

File: sources/platform/integrations/airbyte.md (hunk 3 above)
Comment (TC-MO) [reply sources/platform/integrations/airbyte.md:14]: I fixed the second occurence of Apify dataset , but I feel like first should be left capitalized as it is the name of the connector itself within Airbyte.

**Review by TheoVasilis — APPROVED** [review]

**Pull request merged**